src/core/ui.js

```diff
@@ -59,14 +59,29 @@ respecPill.addEventListener("click", function(e) {
   this.setAttribute("aria-expanded", String(menu.hidden));
   menu.hidden = !menu.hidden;
 });
+document.documentElement.addEventListener("keyup", e => {
+  e.stopPropagation();
+  if (e.keyCode === 73) {
```

Comment:
I think `keyCode` is deprecated - use [`key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) instead.